salesagility/SuiteCRM
# Fix # - Included support for _override_sugarfield on delete vardef

## Description




If you delete the custom variable, the file (custom/Extension/modules/<YourModuleName>/Ext/Vardefs/<YourFieldName_c>.php) won't be deleted.  

## Motivation and Context


## How To Test This


1. Create a custom variable via Studio. 
2. Delete the variable:  
![image](https://github.com/salesagility/SuiteCRM/assets/116086008/f81bc3a9-c792-4f00-923a-3083747284b1)
3. Check the custom folder (mentioned above) to make sure the file is still there. 

## Types of changes


### Final checklist
